src/main/java/org/jabref/gui/keyboard/KeyBinding.java

```diff
@@ -93,27 +93,35 @@
     WEB_SEARCH("Web search", Localization.lang("Web search"), "alt+4", KeyBindingCategory.SEARCH),
     WRITE_XMP("Write XMP", Localization.lang("Write XMP"), "F6", KeyBindingCategory.TOOLS);
 
-    private final String key;
+    private final String constant;
     private final String localization;
     private final String defaultBinding;
     private final KeyBindingCategory category;
 
-    KeyBinding(String key, String localization, String defaultBinding, KeyBindingCategory category) {
-        this.key = key;
+    KeyBinding(String constantName, String localization, String defaultKeyBinding, KeyBindingCategory category) {
+        this.constant = constantName;
         this.localization = localization;
-        this.defaultBinding = defaultBinding;
+        this.defaultBinding = defaultKeyBinding;
         this.category = category;
     }
 
-    public String getKey() {
-        return key;
+    /**
+     * This method returns the enum constant value
+     * @return
+     */
+    public String getConstant() {
```

Comment:
This constant is just the name of the enum value. It is used in several cases to compare bindings and to map between the settings and the enum values dynamically. It is used in KeyBindingsRepositoriy 
I just renamed it, because "key" is ambigous in this content and always confused me.